core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -37,8 +37,8 @@ <h4 class="title-table">Summary</h4>
         </th>
         <th>Disk Used</th>
         <th>Cores</th>
-        <th>Active Tasks</th>
-        <th>Failed Tasks</th>
+        <th><span data-toggle="tooltip" data-placement="top" title="Active Tasks show in shaded blue. % of color opacity range from 0 to maxTasks.">Active Tasks</span></th>
+        <th><span data-toggle="tooltip" data-placement="top" title="Failed Tasks show in shaded red. Color opacity reach max at 10% of Total Tasks.">Failed Tasks</span></th>
```

Comment:
Likewise, maybe:
"Number of tasks that have failed on this executor. Darker shading highlights executors with a high proportion of failed tasks."

Reply:
Yes, they are more suitable，thank you！